projects/vdk-plugins/vdk-oracle/src/vdk/plugin/oracle/ingest_to_oracle.py

```diff
@@ -126,7 +126,13 @@ def _add_columns(self, table_name: str, payload: List[Dict[str, Any]]) -> None:
         all_columns = {
             _normalize_identifier(col) for row in payload for col in row.keys()
         }
-        new_columns = all_columns - existing_columns.keys()
+        diff_columns = all_columns - existing_columns.keys()
```

Comment:
So the values in existing columns are a mix of lower and upper case? 
how did this happen? can we not just have lower in the existing columns ?

Reply:
They're case-insensitive, Oracle will give you upper-case column names if you don't put the column name in quotes when creating the table. However, someone might try to put it in quotes. In this case, we'll get a case-sensitive name and it becomes a mess.